src/main/java/com/puppycrawl/tools/checkstyle/checks/design/HideUtilityClassConstructorCheck.java

```diff
@@ -142,6 +159,26 @@ private static boolean isStatic(DetailAST ast) {
             .findFirstToken(TokenTypes.LITERAL_STATIC) != null;
     }
 
+    /**
+     * Setter to ignore classes annotated with the specified annotation(s).
```

Comment:
please add `Name in this property must be an exact match for the annotation name on the class. If target class has annotation specified as fully qualified name (with package) and by simple name, this property also need to have two values: simple name and fully qualified name.`